benchmarks.yml

```diff
@@ -49,6 +49,8 @@ optcarrot:
   desc: optcarrot is a functional headless NES emulator, run on a specific game cartridge for a specific number of frames.
 rubykon:
   desc: Ruby solver for Go (the boardgame.) Runs 1,000 iterations forward from an initial starting board.
+liquid-c:
+  desc: liquid-c renders a chosen-for-profiling Liquid theme repeatedly.
```

Comment:
I think we should put it in the headline category since it's used on SFR

Reply:
Done